packages/odyssey-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -64,9 +66,9 @@ export const Tooltip = withTheme(
           className={tooltipClasses}
           role="tooltip"
           color={false}
-          font-size={false}
-          font-weight={false}
-          line-height={false}
+          fontSize={false}
```

Comment:
VRT didn't catch this because our Tooltip stories don't test the interaction. I've updated [OKTA-444256](https://oktainc.atlassian.net/browse/OKTA-444256) to capture this.

Reply:
good catch! I'm surprised typescript didn't catch this as props formatted with hyphens aren't part of our public interface for Box! 🤔